### PR TITLE
[Dygraph]rename GridMaskOp to GridMask in operator.py to avoid conflict

### PR DESCRIPTION
**Pull request by wangxinxin08**

### CODE REVIEW

File: ppdet/data/transform/batch_operator.py [hunk 1]
```diff
@@ -31,10 +31,8 @@
 logger = logging.getLogger(__name__)
 
 __all__ = [
-    'PadBatchOp',
-    'Gt2YoloTargetOp',
-    'Gt2FCOSTargetOp',
-    'Gt2TTFTargetOp',
+    'PadBatchOp', 'BatchRandomResizeOp', 'Gt2YoloTargetOp', 'Gt2FCOSTargetOp',
+    'Gt2TTFTargetOp', 'Gt2Solov2TargetOp'
 ]
```
Comment (jerrywgz) [inline ppdet/data/transform/batch_operator.py:36]: 'Gt2TTFTargetOp',

File: ppdet/data/transform/operator.py [hunk 2]
```diff
@@ -224,7 +224,7 @@ def apply(self, sample, context=None):
 
 
 @register_op
-class GridMaskOp(BaseOperator):
+class GridMask(BaseOperator):
```
Comment (yghstill) [inline ppdet/data/transform/operator.py:227]: super(GridMask, self).__init__()

Comment (wangxinxin08) [reply ppdet/data/transform/operator.py:227]: done, thanks

File: ppdet/data/transform/batch_operator.py (hunk 1 above)
Comment (wangxinxin08) [reply ppdet/data/transform/batch_operator.py:36]: done, thanks

**Review by jerrywgz — APPROVED** [review]

**Pull request merged**